packages/embla-carousel/src/components/SlidesToScroll.ts

```diff
@@ -49,7 +49,8 @@ export function SlidesToScroll(
         const gapB = !loop && isLast ? direction(endGap) : 0
         const chunkSize = mathAbs(edgeB - gapB - (edgeA + gapA))
 
-        if (chunkSize > viewSize + pixelTolerance) groups.push(rectB)
+        if (index !== 0 && chunkSize > viewSize + pixelTolerance)
```

Comment:
We can modify this like so:

```suggestion
        if (index && chunkSize > viewSize + pixelTolerance)
```